src/instantiate.ts

```diff
@@ -386,20 +392,53 @@ export async function loadAllofExtension(context: vscode.ExtensionContext) {
               ...clearListArray
             ]
           }
-          starRemoved = false;
         }
+        starRemoved = false;
       })
 
-      quickPick.onDidAccept(() => {
-        const selection = quickPick.selectedItems[0].label;
+      quickPick.onDidAccept(async () => {
+        let selection = quickPick.selectedItems[0].label;
         if (selection && selection !== LOADING_LABEL) {
           if (selection === clearList) {
             storage!.setSourceList({});
+            quickPick.items = clearListArray;
             vscode.window.showInformationMessage(`Cleared list.`);
-            quickPick.hide()
           } else {
             const selectionSplit = selection.split('/')
```

Comment:
Looks like `selectionSplit` doesn't account for case, so if a lowercase value is provided, the rest of this code fails because it expects members in uppercase for the SQL statement.

```suggestion
            const selectionSplit = selection.toUpperCase().split('/');
```

Reply:
The selection will always be in uppercase, if it's a member:
- if the member is entered by the user, the value is put in the list in uppercase
- if the member is from a search, it is in uppercase by SQL
So it should not be necessary to convert to uppercase. We also don't want to convert streamfiles to uppercase.

Reply:
Can you confirm this is user error then? I have all these lowercase listings in my quick pick from prior usage.

<img width="623" alt="image" src="https://github.com/codefori/vscode-ibmi/assets/3708366/f65cac90-23fc-4dfd-8e39-64469d99f893">

Reply:
I also have this enabled, so it looks like it is adding lowercase names to the list! 
<img width="920" alt="image" src="https://github.com/codefori/vscode-ibmi/assets/3708366/95ac565e-abb9-42fb-a73b-ff8abe35974d">

Reply:
Looks like it's this line of code storing the values, including as lowercase when that option is enabled: https://github.com/chrjorgensen/code-for-ibmi/blob/d863157505d93b76388c9c9092fee9c398375394/src/views/objectBrowser.ts#L216

Reply:
The uppercase issue has been fixed.